deployments/dev-local-3nodes/docker-compose.yml

```diff
@@ -82,8 +82,8 @@ services:
       file: ../docker-compose-definitions.yml
       service: medco-unlynx
     ports:
-      - "2002:2002"
-      - "2003:2003"
+      - "2002:2000"
```

Comment:
Is this working? The `configuration/*.toml` files specify the ports that were there before (2002/2003 and 2004/2005). This is what control which port the process is listening too. It is strange that it works with this?

Reply:
Interesting...well, for working, it works...probably there is some technicality in the docker networking that we are missing...how do you control which port the process is listening too? What is the piece of code doing that?

Reply:
Ah yes...I understood what is going on...this is working only because the containers are on the same host, so whatever traffic is sent, say, to 172.31.0.111:2002 is actually redirected to localhost:2002, which is redirected to port 2000 on the container...if the container was not on the same host that would not work, because it is actually listening on port 2000. in the end it does not change anything for a local deployment as this one, I can go back to the previous version, but doesn't it make more sense that all unlynx nodes are listening to the same ports and then let the remapping do its job for cases like this when the containers are on the same host?

Reply:
It is onet, controlled within medco-unlynx/unlynx that does it.
As long as communications are within docker virtual network it's OK, this setting does not have an impact. However when it's not (and it is not for medco-loader with the setting `network_mode: host`), it should be problematic.
What was the reason for modifying this? I'd rather keep it as it was previously unless for good reason.

Reply:
agreed, put them as before